### DISCUSSION TITLE
Section 02: Facing the error TypeError: __new__() missing 1 required positional argument: 'task'

### DISCUSSION THREAD

**Discussion by karthikbhandary2** [mrdbourke/pytorch-deep-learning#318] [Q&A] [answered]
Hello

I am getting the following error. I looked it up on the net and found that I need to pass an argument. I tried giving it the argument as _'multiclass'_ but it is leading to an _**AssertionError**_. Help would be great!
![2023-02-27_07-06-46-f1c74222a19dd2d6916a6e8de12b8e32](https://user-images.githubusercontent.com/34509856/221722338-1dedb65a-05b0-4fb0-93ff-d9a4c398e966.png)

**Comment by PatTheAtak** [comment]
Could you post your code of `Accuracy()`

**Comment by mrdbourke** [comment, marked as answer]
Hey @karthikbhandary2,

Good question!

There's been an update to TorchMetrics since the video was recorded.

Please see the fix here: https://github.com/mrdbourke/pytorch-deep-learning/discussions/200#discussioncomment-4339563

Or alternatively, you can try:

```python
try:
    from torchmetrics import Accuracy
except:
    !pip install torchmetrics==0.9.3 # this is the version we're using in this notebook (later versions exist here: https://torchmetrics.readthedocs.io/en/stable/generated/CHANGELOG.html#changelog)
    from torchmetrics import Accuracy

# Setup metric and make sure it's on the target device
torchmetrics_accuracy = Accuracy(task='multiclass', num_classes=4).to(device)

# Calculate accuracy
torchmetrics_accuracy(y_preds, y_blob_test)
```

**Reply by karthikbhandary2** [reply]
Thank @mrdbourke for the response. I've looked up the #200 fix and I've done exactly as it was told there and my code is working now.
